sherpa/utils/__init__.py

```diff
@@ -119,6 +120,7 @@
 
 
 class NoNewAttributesAfterInit(object):
+
```

Comment:
Is there a style guide that suggests a blank line here? I don't think it should hold back the PR; just wondering whether we want to be consistent with docstrings on classes (which is a larger issue than this PR).